CacheMultimap.adoc

```diff
@@ -46,33 +45,52 @@ public interface CacheMultimap<K, V> {
 
 The underlying implementation will wrap a normal Cache. Some of the suggestions :
 
-== Core Module
 
-[source,java]
-----
-public interface EmbeddedCacheManager {
- ...
- 
- <K, V> CacheMultimap<K, V> getCacheMultimap(String cacheName, boolean createIfAbsent);
-}
+Apparently there is a problem with functional commands.
+They don't really work efficiently over Hot Rod (does get/replace in a loop).
+We would need to add some more handling in the protocol to allow for only partial replication
+of values and only 1 remote call.
 
-----
 
-== Commons Module
+== Embedded Multimap
 
-[source, java]
-----
-public class CacheMultimapFactory {
+A new maven module is created.
 
-   static <K, V> CacheMultimap<K, V> CacheMultimapFactory.get(Cache<K,Object> c) {
-      ...
-   }
-}
-----
+To create a multimap cache
+
+```java
+      EmbeddedCacheManager cm = ...
+      MultimapCacheManager multimapCacheManager = EmbeddedMultimapCacheManagerFactory.from(cm);
```

Comment:
Do we need the `MultimapCacheManager`? Why don't you just pass the underlying cache to the factory?

Reply:
As we don't have cache namespaces, or a naming convention about this, we should probably just go for a shared namespace, and potentially accept access through the traditional cache API

Reply:
@tristantarrant yes, I I said before, this would be the best right now. The PR for the first version is ready !